### PR TITLE
[AC-9634]Show Enterprises on /startups and correct field label on partner forms

### PR DESCRIPTION
**Pull request by AmosWels**
## [AC-9634](https://masschallenge.atlassian.net/browse/AC-9634)

**Changes Introduced**
- Show Enterprises on `/mystartups`
- Correct field labels on EditCompanyProfileForm to match mockup

**Testing**
- Check out `AC-9634`, Run `make migrate`
- As a logged-in user, visit http://localhost:8181/mycompanies/ or https://test-cam.masschallenge.org/mycompanies/
- As a Staff user, visit http://staff.localhost:8181/partner/add/ or https://test-cam.masschallenge.org/partner/add/
- Confirm that the definition of done according to [AC-9634](https://masschallenge.atlassian.net/browse/AC-9634) is satisfied

[Sibling PR](https://github.com/masschallenge/accelerate/pull/3336)

### CODE REVIEW

File: accelerator/migrations/0110_add_verbose_names_on_partner_fields.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Generated by Django 2.2.28 on 2022-07-05 15:41
+
+from django.db import migrations, models
+import embed_video.fields
```
Comment (iankigen) [inline accelerator/migrations/0110_add_verbose_names_on_partner_fields.py:4]: unused import

Comment (AmosWels) [reply accelerator/migrations/0110_add_verbose_names_on_partner_fields.py:4]: embed_video.fields is used [here](https://github.com/masschallenge/django-accelerator/pull/523/files/517f31020bcc925febbf3697d81a1f59bc0d1e51#diff-742f77128930309477aa336c570de0f09c57007df18950a8fbd02230cdfc9839R37)

Comment (iankigen) [reply accelerator/migrations/0110_add_verbose_names_on_partner_fields.py:4]: ah, missed that

**Review by shakirandagire — APPROVED** [review]
The changes look good to me

**Pull request merged**